service/history/workflow/update/util.go

```diff
@@ -39,9 +37,6 @@ import (
 const libraryName = "go.temporal.io/service/history/workflow/update"
 
 type (
-	// lazyOutcome adapts a func to the future.Future[*updatepb.Outcome] interface
```

Comment:
Sorry to see this going...

Reply:
We'll think of another use for it. Let lazyOutcome live on in your heart.